UnitTest++/VxWorks/TimeHelpers.cpp

```diff
@@ -0,0 +1,36 @@
+#include "TimeHelpers.h"
+
+namespace UnitTest {
+
+   Timer::Timer()
+   {
+      m_startTime.tv_sec = 0;
```

Comment:
Yes that can be done but I just implemented it in the same way as done in the Posix version of TimeHelpers.cpp.